src/pages/patientView/timeline2/TimelineWrapperStore.tsx

```diff
@@ -0,0 +1,81 @@
+import { action, observable } from 'mobx';
+import _ from 'lodash';
+import * as React from 'react';
+import { Mutation } from 'cbioportal-ts-api-client';
+import { MutationStatus } from '../mutation/PatientViewMutationsTabUtils';
+
+export default class TimelineWrapperStore {
+    @observable groupByOption: Readonly<string> | null = null;
```

Comment:
how come you are using ReadOnly here and elsewhere?

Reply:
Correct. It was left after a copy paste of another observable. Removed it.